department-of-veterans-affairs/caseflow
# CASEFLOW-1647 ScheduleHearingTask Alert for substitutions

Resolves [Caseflow-1647](https://vajira.max.gov/browse/CASEFLOW-1647)

### Description
When a COB User proceeds down the substitution path and makes it to the "task selection" to copy from the original appeal to the substitution appeal - if they select a "Schedule Hearing" task, an alert should display saying as per this Figma:

<img width="447" alt="2021-07-23 15_20_11-Granted Substitutions 2021 – Figma" src="https://user-images.githubusercontent.com/23639439/126831117-41b415dc-0381-4ad0-9191-bc88ff5ea125.png">

### Acceptance Criteria
- [ ] When user selects "Schedule Hearing" Task, an informational banner should appear.
- [ ] Task logic for the selection of the tasks themselves are being handled in [Caseflow-1499](https://vajira.max.gov/browse/CASEFLOW-1499)

### Testing Plan
1. Login as "COB_USER"
2. Search cases for "54545454"
3. Select the "Hearing" docket appeal.
4. Click on "Add Substitute", populate the substitution granted date, select a substitute, then continue.
5. Verify that there is a "Schedule hearing" task that is unchecked.
6. Select the task and the alert banner should appear.
7. Select the task again and the banner should be gone.
8. Do not need to continue any further; the logic handling beyond displaying the banner is in another story.

### User Facing Changes

**With the Alert:**
<img width="677" alt="WithScheduleHearingTaskAlert" src="https://user-images.githubusercontent.com/23639439/126831842-d8405410-1540-4979-9ac9-18c3fb8ad396.png">

**Without the Alert:**
<img width="657" alt="WithoutScheduleHearingTaskAlert" src="https://user-images.githubusercontent.com/23639439/126831795-d7779fb5-0df2-4f19-883c-83a201edc67c.png">



### Storybook Story
[] Storybook component has been created and added to this story.
